sales/sale_amazon/apply.rst

```diff
@@ -0,0 +1,148 @@
+
+    [Description of your organization's policy regarding Amazon Information]
+
+- **List all non-Amazon MWS sources where you retrieve Amazon Information.**
+
+  Odoo only relies on MWS to retrieve Amazon Information.
+
+- **Describe how your organization restricts public access to databases, file servers, and
+  desktop/developer endpoints.**
+
+  | - Access to the postgreSQL database through HTTP is disabled and standard ports are closed. The database is only accessible through a socket on the server itself.
+  | - The reverse proxy only serves whitelisted directories that are only from sources controlled by Odoo S.A.
+  | - API endpoints are password protected (PBKDF2 & SHA512 encryption, salted, and stretched for thousands of rounds).
+  | - Login credentials are always transmitted securely over HTTPS.
+  |
+
+- **Describe how your organization uniquely identifies employees and restricts access to Amazon
+  Information on a need-to-know basis.**
+
+  - If all your employees are properly assigned separate users and given only relevant access rights:
+
+    Access rights are provided to employees based on their role within the company and are
+    progressive based on their responsibility.
+
+    For example, salespersons only have access to their own leads/quotes (and thus no access to
+    quotes generated through the Amazon API). A salesmanager has access to all quotes/leads for
+    reporting purposes (including quotes generated through the Amazon API). A quote will generate a
+    delivery order which will be accessible to a 'normal' user of the Inventory application for him
+    to be able to print the delivery label and pack the products.
+
+  - If your employees share users or if they are given more rights than needed:
+
+    [Description of your organization's policy for the assignation of users and access rights to
+    your employees]
+
+- **Describe how your organization prevents Amazon Information from being accessed from employee
+  personal devices.**
+
+  Odoo does not prevent employees from accessing the organization's data from personal devices.
+  Role-based restrictions and access rights still apply.
+
+- **Provide details on your organization's privacy and data handling policies (a link to your policy
+  is also acceptable).**
+
+  [Description of your organization's privacy and data handling policies]
+
+- **Describe where your organization stores Amazon Information and provide details on how you
+  encrypt this information (e.g., algorithm).**
+
+  Amazon Information is stored in an unencrypted database. Direct access to the database is not
+  possible for the customer outside of UI interactions or API calls. Granular access rights control
+  ensures that access is not shared to all users of the database.
+
+- **Describe how your organization backups or archives Amazon Information and provide details on how
+  you encrypt this information (e.g., algorithm).**
+
+  Backups are stored in an unencrypted database.
```

Comment:
The entire database is backed up once a day and backups are kept for a minimum of three months according to the [Odoo Online SLA](https://www.odoo.com/cloud-sla). Backups are hosted on several remote servers as unencrypted database dumps; these backups can only be retrieved by Odoo S.A. employees through support requests.